### ISSUE TITLE
Bring design in line with playbook refresh

### ISSUE THREAD

**Issue by yndajas** [dxw/accessibility-manual#143]
The accessibility manual was initially largely a clone of the playbook (see #1), copying the styles from circa October 2022. Various playbook PRs have refreshed the design since then, including but probably not limited to:
- https://github.com/dxw/playbook/pull/990
- https://github.com/dxw/playbook/pull/993
- https://github.com/dxw/playbook/pull/998
- https://github.com/dxw/playbook/pull/1204

We should probably bring most of these enhancements into the accessibility manual. We may want to use a different colour scheme (still using the dxw brand colours) for the accessibility manual so that it retains its own identity. This work should be done in collaboration with interaction designers

**Comment by yndajas** [comment]
> We may want to use a different colour scheme (still using the dxw brand colours) for the accessibility manual so that it retains its own identity.

@mingoland suggests we change the top-left corner from navy to blue (from our brand colours): #004876
